prettier.config.js

```diff
@@ -1,55 +1,93 @@
-// Modified from https://github.com/matzkoh/prettier-plugin-packagejson to use the custom sort-package.json package
-function prettier_plugin_packagejson() {
-  function requireSafe(path) {
-    try {
-      return require(path)
-    } catch (error) {
-      // no operations
+
+const plugins = [ require('prettier-plugin-jsdoc') ];
+
+
+/*
+ *  Modified from https://github.com/matzkoh/prettier-plugin-packagejson 
+ *  to use the custom sort-package.json package
+ */
+
+{
+    
+    function requireSafe(path){
+        try { return require(path); }
+        catch () {}
+    }
+    
+    
+    //  Static
+    
+    const isPackageJSON = /(^|\\|\/)package\.json$/;
+
+    const
+        parser_babylon = 'prettier/parser-babylon' ,
+        parser_babel = 'prettier/parser-babel' ,
+        package_sort = 'sort-package-json' ;
+    
```

Comment:
```js
const parser_babylon = 'prettier/parser-babylon';
cosnt parser_babel = 'prettier/parser-babel';
const package_sort = 'sort-package-json';
```

You call this more readable?

Reply:
I see what you mean in that it could be called less readable, in that there is more `const` that adds to clutter. But to me at least, it is clearer what each lines' intent is with the `const` keyword in the beginning of each line clearly saying that it is a const declaration. Also each variable name is still aligned with the others, so theres no problem there

Reply:
@aminya The only thing it warned me about is a non-existant `)` in the `.eslintrc` file.